src/client/containers/EventsContainer.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import PropTypes from 'prop-types'
+
+import Grid from '@material-ui/core/Grid'
+import { withStyles } from '@material-ui/core/styles'
+
+import EventsView from '../pages/Events/EventsView'
+
+/*
+*
+* FAKE API HANDLING PART - START
+*/
+// Fake Events API
+import { EventsResponse, EventField, ProvidedTimes } from './EventsFakeAPI'
+
+function getEventsAsArray(givenTime) {
+  const eventsAsArray = [];
+  const eventsForGivenTime = EventsResponse[givenTime];
+
+  if (EventsResponse[givenTime]) {
+    for (let eventIdx = 0; eventIdx < eventsForGivenTime.length; eventIdx += 1) {
+      const event = eventsForGivenTime[eventIdx];
+      const eventArr = []
+
+      for (const field of EventField) {
+        eventArr.push(event[field])
+      }
+
+      eventsAsArray.push(eventArr)
+    }
+  }
+  return eventsAsArray;
+}
+
+function renderEventsViews(givenTime) {
```

Comment:
This logic should be in another component that is maybe called `EventSection` that handles the rendering of events in the right place. This should not be a part of the Fake API. This should be a real component.

Reply:
Actually, `getEventsAsArray` and `renderEventsViews` should be one larger component